tests/Admin/AdminTest.php

```diff
@@ -948,6 +948,9 @@ public function testGetExtensions(): void
         $admin->addExtension($adminExtension1);
         $admin->addExtension($adminExtension2);
         static::assertSame([$adminExtension1, $adminExtension2], $admin->getExtensions());
+
+        $admin->removeExtension($adminExtension2);
+        static::assertSame([$adminExtension1], $admin->getExtensions());
```

Comment:
Would be nice if we also check for removing non existent extension

Reply:
Done